docker-compose.yml

```diff
@@ -1,22 +1,24 @@
 version: '2'
 
 services:
+  mongodb:
+    image: 'bitnami/mongodb'
```

Comment:
Maybe we could add here the tag `latest` for consistence with the node entry. What do you think?

Reply:
👍